app/services/analytics_events.rb

```diff
@@ -1306,9 +1306,111 @@ def idv_doc_auth_submitted_pii_validation(
+  # @option proofing_results [Hash] context[:stages][:threatmetrix] Object holding details about the call made to the device profiling vendor
+  # @option proofing_results [String] context[:stages][:threatmetrix][:client] Identifier string indicating which client was used.
+  # @option proofing_results [Boolean] context[:stages][:threatmetrix][:success] Whether the request to the vendor succeeded.
+  # @option proofing_results [Hash] context[:stages][:threatmetrix][:errors] Hash describing errors encountered when making the request.
+  # @option proofing_results [String,nil] context[:stages][:threatmetrix][:exception] If an exception was encountered making the request to the vendor, its message is provided here.
+  # @option proofing_results [Boolean] context[:stages][:threatmetrix][:timed_out] Whether the request to the vendor timed out.
+  # @option proofing_results [String] context[:stages][:threatmetrix][:transaction_id] Vendor-specific transaction ID for the request.
+  # @option proofing_results [Hash] context[:stages][:threatmetrix][:response_body] JSON body of the response returned from the vendor. PII has been redacted.
+  # @option proofing_results [String] context[:stages][:threatmetrix][:response_body][:account_lex_id] LexID associated with the response.
+  # @option proofing_results [String] context[:stages][:threatmetrix][:response_body][:session_id] Session ID associated with the response.
+  # @option proofing_results [String] context[:stages][:threatmetrix][:review_status] One of "pass", "review", "reject".
+  # @param skip_hybrid_handoff [Boolean] Whether the user should skip hybrid handoff (i.e. because they are already on a mobile device)
+  # @param ssn_is_unique [Boolean] Whether another Profile existed with the same SSN at the time the profile associated with the current IdV session was minted.
+  # @param step [String] Always "verify" (leftover from flow state machine days)
+  # @param success [Boolean] Whether identity resolution succeeded overall
+  def idv_doc_auth_verify_proofing_results(
+    ab_tests: nil,
+    acuant_sdk_upgrade_ab_test_bucket: nil,
+    address_edited: nil,
+    address_line2_present: nil,
+    analytics_id: nil,
+    errors: nil,
+    flow_path: nil,
+    irs_reproofing: nil,
+    lexisnexis_instant_verify_workflow_ab_test_bucket: nil,
+    opted_in_to_in_person_proofing: nil,
+    proofing_results: nil,
+    skip_hybrid_handoff: nil,
+    ssn_is_unique: nil,
+    step: nil,
+    success: nil,
+    **extra
+  )
+    track_event(
+      'IdV: doc auth verify proofing results',
+      ab_tests:,
+      acuant_sdk_upgrade_ab_test_bucket:,
+      address_edited:,
+      address_line2_present:,
+      analytics_id:,
+      errors:,
+      flow_path:,
+      irs_reproofing:,
+      lexisnexis_instant_verify_workflow_ab_test_bucket:,
+      opted_in_to_in_person_proofing:,
+      proofing_results:,
+      skip_hybrid_handoff:,
+      ssn_is_unique:,
+      step:,
+      success:,
+      **extra,
```

Comment:
we might consider doing the `{}.compact` trick here just in case some of these keys are not present in every case?
```suggestion
      'IdV: doc auth verify proofing results',
      **{
        ab_tests:,
        acuant_sdk_upgrade_ab_test_bucket:,
        address_edited:,
        address_line2_present:,
        analytics_id:,
        errors:,
        flow_path:,
        irs_reproofing:,
        lexisnexis_instant_verify_workflow_ab_test_bucket:,
        opted_in_to_in_person_proofing:,
        proofing_results:,
        skip_hybrid_handoff:,
        ssn_is_unique:,
        step:,
        success:,
        **extra,
      }.compact,
```

Reply:
Yes, I think I prefer this (I would be in favor of doing this globally, for all events). I'll add and update specs